public/app/features/migrate-to-cloud/onprem/Page.tsx

```diff
@@ -70,7 +70,9 @@ const PAGE_SIZE = 50;
 function useGetLatestSnapshot(sessionUid?: string, page = 1) {
   const [shouldPoll, setShouldPoll] = useState(false);
 
-  const listResult = useGetShapshotListQuery(sessionUid ? { uid: sessionUid } : skipToken);
+  const listResult = useGetShapshotListQuery(
+    sessionUid ? { uid: sessionUid, page: 1, limit: 1, latest: true } : skipToken
```

Comment:
Would it be clearer to pass something like sort="latest"?

Reply:
I opted on that for simplicity, since there is no other sorting option needed at the moment. If we think this might expand, yes sure can modify

Reply:
Can we expand in the future without breaking older versions or having to support more than one parameter? If so that's fine